Pipfile

```diff
@@ -10,31 +10,33 @@ cryptography35_linux = { path='./wheelhouse/cryptography-2.3.dev1-cp35-cp35m-lin
 cryptography36_linux = { path='./wheelhouse/cryptography-2.3.dev1-cp36-cp36m-linux_x86_64.whl', markers="python_version >= '3.6' and python_version < '3.7' and platform_system == 'Linux'"}
 cryptography36_darwin = { path='./wheelhouse/cryptography-2.3.dev1-cp36-cp36m-macosx_10_13_x86_64.whl', markers="python_version >= '3.6' and python_version < '3.7' and platform_system == 'Darwin'"}
 
-requests = "*"
-maya = "*"
-hendrix = "==3.0.0"
+# NuCypher Managed Packages #
 constantSorrow = {git = "https://github.com/nucypher/constantSorrow.git", ref = "nucypher-depend"}
 bytestringSplitter = {git = "https://github.com/nucypher/byteStringSplitter.git", ref = "nucypher-depend"}
-sqlalchemy = "*"
-apistar = "*"
+pyumbral = {git = "https://github.com/nucypher/pyumbral.git", ref = "nucypher-depend"}
 rpcudp = {git = "https://github.com/nucypher/rpcudp", ref = "nucypher-depend"}
 kademlia = {git = "https://github.com/nucypher/kademlia", ref = "nucypher-depend"}
-pynacl = "*"
-pysha3 = "*"
-py_ecc = "*"
-pyumbral = {git = "https://github.com/nucypher/pyumbral.git", ref = "nucypher-depend"}
-web3 = "*"
+hendrix = "==3.0.0"
+
+# 3rd Party Packages #
+requests = "*"
+maya = "*"
+sqlalchemy = "*"
+apistar = "*"
+web3 = "==4.4.1"
 
 [dev-packages]
+
+# Pytest Packages #
 pytest = "*"
-mypy = "*"
+pytest-twisted = "*"
 pytest-mypy = "*"
-codecov = "*"
-coverage = "*"
 pytest-cov = "*"
+
+# Ethereum Dev Packages #
 py-solc = "*"
-py-evm = "*"
-eth-tester = "*"
+py-evm = "==0.2.0a18"
```

Comment:
Wait.. are you sure?

Reply:
We are still being effected by a `evm.exceptions.ValidationError: mix hash mismatch; 0x...` when running `py-evm<=0.2.0a18`.  Similar to the Issue seen here:  https://github.com/ethereum/eth-tester/issues/99 - except that `eth-tester` seemingly does support newer versions of `py-evm`: https://github.com/ethereum/eth-tester/blob/master/setup.py#L32.

I'm unsure what the fix is at this time.